packages/neuron-wallet/src/services/asset-account-service.ts

```diff
@@ -256,14 +256,23 @@ export default class AssetAccountService {
       .values(sudtTokenInfoEntity)
       .onConflict(`("tokenID") DO NOTHING`)
       .execute()
-
-    await getConnection()
+    const existAccountAcount = await getConnection()
+      .getRepository(AssetAccountEntity)
       .createQueryBuilder()
-      .insert()
-      .into(AssetAccountEntity)
-      .values(assetAccountEntity)
-      .onConflict(`("tokenID", "blake160") DO NOTHING`)
-      .execute()
+      .where({
+        tokenID,
+        blake160
+      })
+      .getCount()
+    // check whether the entity exists before insert. Reason: https://github.com/Magickbase/neuron-public-issues/issues/184#issue-1749746997
+    if (!existAccountAcount) {
+      await getConnection()
+        .createQueryBuilder()
+        .insert()
+        .into(AssetAccountEntity)
+        .values(assetAccountEntity)
+        .execute()
```

Comment:
I'm not very familiar with TypeORM, but can we directly use `await getConnection().getRepository(SudtTokenInfoEntity).upsert` here instead of making two separate queries?
In theory, the database should support such an operation, and it might be a matter of how to achieve this through TypeORM's API calls.

Reply:
More detail can be found https://github.com/Magickbase/neuron-public-issues/issues/184#issue-1749746997, it may be a bug of `typeorm`